README.md

```diff
@@ -0,0 +1,74 @@
+# Console for Apache Kafka™ on Kubernetes
+
+This project is a web console designed to facilitate interactions with Apache Kafka™ instances on Kubernetes, leveraging the [Strimzi](https://strimzi.io) Cluster Operator. 
+It is composed of two main parts: 
+
+- a REST API backend developed with Java Quarkus
+- a user interface (UI) built with Next.js and  [PatternFly](https://patternfly.org)
+
+### Getting started
+
+#### API
+
+Ensure you have Kubernetes and Strimzi Cluster Operator installed on your system.
+
+```bash
+cd api
+```
+
+Create a `.env` file containing the details about how the API should connect to the Kafka cluster.
+For a Kafka cluster installed using [Strimzi's Quick Starts](https://strimzi.io/quickstarts/) it should look like this:
+
+```.dotenv
+CONSOLE_KAFKA_MY_CLUSTER=kafka/my-cluster
+CONSOLE_KAFKA_MY_CLUSTER_BOOTSTRAP_SERVERS=my-cluster-kafka-bootstrap:9092
+```
+
+Then run the application. 
+
+```bash
+./mvn quarkus:dev -DskipTests
+```
+
+This should result in Quarkus starting up, with the REST APIs available on localhost port 8080:
+
+* [API documentation](http://localhost:8080/swagger-ui)
+
+#### UI
+
+```bash
+cd ui
+```
+
+Create a `.env` file containing the details about where to find the API server, and some additional config.
+
+```.dotenv
+BACKEND_URL=http://localhost:8080
+CONSOLE_METRICS_PROMETHEUS_URL=http://localhost:9090
```

Comment:
what is this? is it the Quarkus server as well? or something else?

Reply:
It's something else, which is installed if the instructions in the `install` folder are followed. It is a current pain-point of developing the UI, we use a dev cluster where all is installed and available but for outside contributors they have to run it on their own env.